### PR TITLE
Added race variants

### PR DESCRIPTION
**Pull request by Villager1**
Added PvP and Manhunt variants

### CODE REVIEW

File: src/main/java/io/github/hielkemaps/racecommand/race/Race.java [hunk 1]
```diff
@@ -46,18 +52,44 @@ public Race(UUID owner, String name) {
...
         sendMessage(Main.PREFIX + "Starting race...");
 
+        for(UUID uuid : players) {
+            Player player = Bukkit.getPlayer(uuid);
+            player.removeScoreboardTag("hunter");
+            player.removeScoreboardTag("runner");
+        }
+
+        //Set hunter for manhunt races
+        if(variantType() == "manhunt") {
```
Comment (aimiebell) [inline src/main/java/io/github/hielkemaps/racecommand/race/Race.java:72]: I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [ES: Comparison of String objects using == or != (ES_COMPARING_STRINGS_WITH_EQ)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#es-comparison-of-string-objects-using-or-es-comparing-strings-with-eq).
This code compares java.lang.String objects for reference equality using the == or != operators. Unless both strings are either constants in a source file, or have been interned using the String.intern() method, the same string value may be represented by two different String objects. Consider using the equals(Object) method instead.

File: src/main/java/io/github/hielkemaps/racecommand/race/Race.java [hunk 2]
```diff
@@ -46,18 +52,44 @@ public Race(UUID owner, String name) {
...
+            int hunterRandomizer = (int) Math.random() * players.size();
```
Comment (aimiebell) [inline src/main/java/io/github/hielkemaps/racecommand/race/Race.java:73]: I detect that this code is problematic. According to the [Correctness (CORRECTNESS)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#correctness-correctness), [RV: Random value from 0 to 1 is coerced to the integer 0 (RV_01_TO_INT)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#rv-random-value-from-0-to-1-is-coerced-to-the-integer-0-rv-01-to-int).
A random value from 0 to 1 is being coerced to the integer value 0. You probably want to multiply the random value by something else before coercing it to an integer, or use the Random.nextInt(n) method.

Comment (Mary904) [inline src/main/java/io/github/hielkemaps/racecommand/race/Race.java:73]: Our static analysis engine detects that your code is problematic. According to the [Correctness (CORRECTNESS)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#correctness-correctness), [RV: Random value from 0 to 1 is coerced to the integer 0 (RV_01_TO_INT)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#rv-random-value-from-0-to-1-is-coerced-to-the-integer-0-rv-01-to-int),

A random value from 0 to 1 is being coerced to the integer value 0. You probably want to multiply the random value by something else before coercing it to an integer, or use the `Random.nextInt(n)` method.

Here, consider to use `(int) (Math.random() * players.size())`.

**Comment by HielkeMaps** [comment]
Thanks for the pull request, I implemented the race variants already. (more object oriented)

**Pull request closed without merging**